### PR TITLE
game_list: Minor changes

### PR DESCRIPTION
**Pull request by lioncash**
Each commit should be fairly self-explanatory.

### CODE REVIEW

File: src/citra_qt/game_list.h [hunk 1]
```diff
@@ -34,7 +34,7 @@ class GameList : public QWidget {
     void LoadInterfaceLayout();
 
 public slots:
```
Comment (yuriks) [inline src/citra_qt/game_list.h]: Do we still need these to be slots if we're using the new connection syntax?

Comment (lioncash) [inline src/citra_qt/game_list.h]: Nah, the new syntax lets you bind regular functions as a slot. I can make them all private functions if that's preferable.

Comment (yuriks) [inline src/citra_qt/game_list.h]: Might as well.

Comment (lioncash) [inline src/citra_qt/game_list.h]: Done

**Pull request merged**